auth/api/iam/client.go

```diff
@@ -80,3 +81,39 @@ func (hb HTTPClient) OAuthAuthorizationServerMetadata(ctx context.Context, webDI
 
 	return &metadata, nil
 }
+
+// PresentationDefinition retrieves the presentation definition for the given web DID and scope(s).
+// We pass the endpoint url for the presentation definition endpoint because we already retrieved the metadata in a previous step.
+// The scopes are evaluated as raw query params and encoded if needed.
```

Comment:
This assumes very tight coupling with the calling code ("previous step"). And clients always handle input encoding, so no need to explicitly that that.

```suggestion
// PresentationDefinition retrieves the presentation definition from the presentation definition endpoint (as specified by RFC021) for the given scope.
```